app/assets/javascripts/directives/require_file.js

```diff
@@ -0,0 +1,24 @@
+ManageIQ.angular.app.directive('requireFile', ['$parse', function ($parse) {
```

Comment:
Do I understand this directive correctly, that it parses uploaded file to string? If so, please move the logic to controller. Basically using $parse and accessing element directly is bad idea, angular should divert you from DOM itself and you should think in different way, because messing with DOM elements make it harded for angular to render and is slower.